app/models/entry/finder.rb

```diff
@@ -70,6 +70,10 @@ def total
       @total ||= entries.sum(:minutes)
     end
 
+    def collaborators
+      @collaborators ||= @entries.map(&:user_id).uniq.map { |user_id| User.find user_id }
```

Comment:
But this will probably fail if empty array will be passed, maybe

    User.find_by(id: @entries.pluck(:user_id))